about/index.html

```diff
@@ -12,11 +12,10 @@
 		{% endcapture %}
 		{{ about_farset | unindent | markdownify }}
 	<hr/>
-<h2>Current Officers</h2>
+<h2>Current Management Board</h2>
 
 <h3 id="directors">Directors</h3>
 <p>Farsets Directors are legally, fiscally, and socially responsible for the strategic health and operation of the Labs</p>
-<p>They are collectively, ultimately responsible for the growth and health of the organisation and it’s community, but are primarily concerned with the financial, legal and strategic running of the organisation.<p>
```

Comment:
It's repeated on three other occasions across the ops manual, coc, articles of association etc etc. It's repetitious and boring.